pycyphal/transport/can/_can.py

```diff
@@ -253,6 +254,18 @@ def make_tracer() -> CANTracer:
         """
         return CANTracer()
 
+    async def spoof_frames(self, frames: typing.Sequence[DataFrame]) -> None:
+        async with self._media_lock:
+            if self._maybe_media is None:
+                raise pycyphal.transport.ResourceClosedError(f"{self} is closed")
+            await self._maybe_media.send(
+                [Envelope(f, loopback=False) for f in frames],
+                monotonic_deadline=asyncio.get_running_loop().time() + 1.0,
```

Comment:
Do we not want to expose the transmission deadline as an argument for this method? I think it would make the logic in Yukon more explicit.